index.html

```diff
@@ -9,691 +9,80 @@
     dt {color: black}
     </style>
       <link href="https://www.w3.org/StyleSheets/TR/2016/W3C-ED" rel="stylesheet" type="text/css">
+    <script src="http://dev.w3.org/2009/dap/ReSpec.js/js/respec.js" class="remove"></script>
```

Comment:
```suggestion
    <script
      src="https://www.w3.org/Tools/respec/respec-w3c"
      class="remove"
      defer
    ></script>
```

This is a very old version of ReSpec. The generated document won't conform to the current W3C Pubrules.

Reply:
Fixed.